official/vision/modeling/factory.py

```diff
@@ -263,8 +263,8 @@ def build_retinanet(
     l2_regularizer: Optional[tf_keras.regularizers.Regularizer] = None,
     backbone: Optional[tf_keras.Model] = None,
     decoder: Optional[tf_keras.Model] = None,
-    num_anchors_per_location: int | dict[str, int] | None = None,
-    anchor_boxes: Mapping[str, tf.Tensor] | None = None,
+    num_anchors_per_location: Union[int, Dict[str, int], None] = None,
```

Comment:
Can this `Union[int, Dict[str, int], None]` be `Optional[Union[int, Dict[str, int]]]`